app/controllers/articles_controller.rb

```diff
@@ -35,34 +35,42 @@ def new
     @article = if @tag.present? && @user&.editor_version == "v2"
                  authorize Article
                  submission_template = @tag.submission_template_customized(@user.name).to_s
-                 Article.new(body_markdown: submission_template.split("---").last.to_s.strip, cached_tag_list: @tag.name,
-                             processed_html: "", user_id: current_user&.id, title: submission_template.split("title:")[1].to_s.split("\n")[0].to_s.strip)
+                 Article.new(body_markdown: submission_template.split("---").last.to_s.strip,
```

Comment:
I would put all params on a new line, so:

```ruby
                 Article.new(
                  body_markdown: submission_template
```

instead of

```ruby
                 Article.new(body_markdown: submission_template
```

what do you think? This way they'll all be aligned, instead of having all but on on the same column.

We use the first option in lots of places in the code too, see also your change in this file on line 72 which is like that

Reply:
Ok, I (think I) have made the changes you requested. I have indented one level for the line continuations; my preference is 2 levels, which I have been doing for years, to differentiate expression continuations from logical nesting levels; but very few people know about or prefer it, so I used 1 level. As an aside, I think that some changes in commits are made by Rubocop and not the committer, unless all Rubocop complaints have been resolved before committing (which I did for this commit).